### ISSUE TITLE
Server Down but HA showing 'Connected' on all cams.

### ISSUE THREAD

**Issue by floodwayprintco** [elad-bar/ha-blueiris#212]
My BlueIris server went down last night. I had some automations set up based on the 'connected' status of cameras, and availability of the server device. They are all showing connected even though I can't pull up any cameras from HA and the server has been down about 8 hours at this point. 

I must be missing something here,  is there a reason my 'connected' status on cameras is not updating?

**Comment by kramttocs** [comment]
The 'connected' status is identifying if Blue Iris is able to connect to the camera not if HA can connect to BI/Camera. So BI has to be running to change this state. Hope that makes sense. 
Additionally, the connected status needs to be setup via MQTT (see the manual for details).
Thanks

**Comment by floodwayprintco** [comment]
Thank you, I think I am missing the MQTT portion. I forgot I set that up manually for the motion.

**Comment by floodwayprintco** [comment]
Thank you. I did find it off that BlueIris had Watchdog settings per camera, but not for the server itself.

I do speculate that the next time I have this issue, that a camera or two would disconnect before the entire system locks up. 

I looked at [Blue Iris Tools](https://ipcamtalk.com/threads/blue-iris-tools-weather-overlay-watchdog-more.190/) but at first glance it doesn't have the MQTT integration, which I want to use to tell HomeAssistant to physically reset the power on the machine with a smart plug.
